### PR TITLE
Remove many Intel references

### PR DESCRIPTION
**Pull request by jgrund**
Signed-off-by: Joe Grund <jgrund@whamcloud.io>

### CODE REVIEW

File: chroma-manager/chroma-manager.spec [hunk 1]
```diff
@@ -20,8 +20,8 @@ License: Proprietary
 Group: Development/Libraries
 BuildRoot: %{_tmppath}/%{name}-%{version}-%{release}-buildroot
 Prefix: %{_prefix}
-Vendor: Intel Corporation <hpdd-info@intel.com>
-Url: http://lustre.intel.com/
+Vendor: whamCloud <iml@whamcloud.com>
```
Comment (johnsonw) [inline chroma-manager/chroma-manager.spec]: Is the casing correct here?

File: chroma-manager/setup.py [hunk 2]
```diff
@@ -14,21 +14,21 @@
...
-    author = "Intel Corporation",
-    author_email = "hpdd-info@intel.com",
-    url = 'http://lustre.intel.com/',
-    license = 'Proprietary',
-    description = 'The Intel® Manager for Lustre* software Monitoring and Administration Interface',
-    long_description = open('README.txt').read(),
-    packages = find_packages(exclude=excludes) + [''],
+    name='chroma-manager',
+    version=PACKAGE_VERSION,
+    author="whamCloud",
```
Comment (johnsonw) [inline chroma-manager/setup.py]: Is this casing correct?

**Comment by johnsonw** [comment]
Looks good, just a couple questions regarding the casing of `whamCloud`.

File: chroma-manager/chroma-manager.spec (hunk 1 above)
Comment (jgrund) [reply chroma-manager/chroma-manager.spec]: as far as I know, yes.

File: chroma-manager/setup.py (hunk 2 above)
Comment (jgrund) [reply chroma-manager/setup.py]: as far as I know, yes.

**Pull request closed without merging**